### DISCUSSION TITLE
W3 codev

### DISCUSSION THREAD

**Discussion by TimAshe77AI** [community/community#136087] [Copilot Conversations]
### Select Topic Area

General

### Body

Some where I picked up copyrighted  material  or lsome it say I had to see github to fix this sry bout subjects  picked I'm not a coder

**Comment by thanhdatpb** [comment]
Accept

**Reply by TimAshe77AI** [reply]
How long you been doing this what is this place ..I'm I member of this

**Comment by TimAshe77AI** [comment]
Am I a member of this ..what is this place why do meet here is this a club or something

**Comment by Akash1134** [comment]
Hi @TimAshe77AI ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as this one. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
